packages/components/pie-button/test/visual/pie-button.spec.ts

```diff
@@ -1,52 +1,30 @@
 import { test } from '@sand4rt/experimental-ct-web';
 import percySnapshot from '@percy/playwright';
+import { PropObject, Combination, getAllPropCombinations } from '@justeattakeaway/pie-webc-core/src/test-helpers/get-all-prop-combos.ts';
```

Comment:
playwright breaks if we use directory imports and it seemed completely broken if I didn't include the file extension either